fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexMaintenanceTest.java

```diff
@@ -576,14 +576,21 @@ void chaosMergeAndUpdateTest() throws InterruptedException, IOException {
 
             // busy merge
             for (int i = 0; i < countReps; i++) {
```

Comment:
This merges in a very tight loop. Maybe adding some delay here to ease the load on the cluster?